drivers/volume/ocp/ocp.go

```diff
@@ -0,0 +1,102 @@
+package ocp
+
+import (
+	"fmt"
+	"github.com/libopenstorage/openstorage/api"
+	torpedovolume "github.com/portworx/torpedo/drivers/volume"
+	"github.com/portworx/torpedo/drivers/volume/portworx"
+	"github.com/portworx/torpedo/drivers/volume/portworx/schedops"
+	"github.com/portworx/torpedo/pkg/errors"
+	"github.com/portworx/torpedo/pkg/log"
+)
+
+const (
+	OcpDriverName = "ocp"
+	// OcpServiceName is the name of the ibm storage driver implementation
+	OcpServiceName = ""
+)
+
+// Provisioners types of supported provisioners
+var provisionersForOcp = map[torpedovolume.StorageProvisionerType]torpedovolume.StorageProvisionerType{
```

Comment:
Can you do it for rgw and cephfs as well?

Reply:
added for rbd and cephfs